config/development.json

```diff
@@ -4,7 +4,10 @@
 		"max_static_paths": 1
 	},
 	"dev_dot": {
-		"max_static_paths": 1
+		"max_static_paths": 1,
+		"algolia": {
+			"tutorialsIndexName": "staging_LEARN"
```

Comment:
💯 I dig the use of the staging index for local dev and preview 👍

Reply:
I changed it up to just use staging for local at the moment. Since we don't have a workflow that auto-updates staging I don't want the author previews to have any weird behavior if the index isn't up to date.